manifests/master/pdf_stack.pp

```diff
@@ -28,13 +28,7 @@
     require             => Yumrepo['robert-gcj'],
   }
 
-  # Regression in 0.12.4, https://github.com/wkhtmltopdf/wkhtmltopdf/issues/3241
```

Comment:
Also, this is tangentially relevant because of the version of the package to cache